src/custom/components/CowSubsidyModal/SubsidyTable.tsx

```diff
@@ -107,7 +114,7 @@ const SubsidyTr = styled.tr<{ selected?: boolean; darkMode?: boolean }>`
     }
     
     > td {
-      color: ${darkMode ? theme.orange : theme.text1};
+      color: ${darkMode ? theme.text1 : theme.text1};
```

Comment:
Now both are identical. Unless you intend to change, you can:

```
      color: ${theme.text1};
```

Reply:
Yes correct. For now intend to keep for a future change.